deploy/ethregistrar/04_deploy_bulk_renewal.ts

```diff
@@ -0,0 +1,56 @@
+import { ethers } from 'hardhat'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { keccak256 } from 'js-sha3'
+import { namehash } from 'ethers/lib/utils';
+
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments, network } = hre
+  const { deploy } = deployments
+  const { deployer, owner } = await getNamedAccounts()
+
+  if (!network.tags.use_root) {
+    return true
+  }
+
+  const root = await ethers.getContract('Root', await ethers.getSigner(owner))
+  const registry = await ethers.getContract('ENSRegistry', await ethers.getSigner(owner))
+  const resolver = await ethers.getContract('PublicResolver', await ethers.getSigner(owner))
+  const registrar = await ethers.getContract('BaseRegistrarImplementation')
+  const controller = await ethers.getContract('ETHRegistrarController')
+
+  const bulkRenewal = await deploy('BulkRenewal', {
+    from: deployer,
+    args: [registry.address],
+    log: true,
+  })
+
+  console.log('Temporarily setting owner of eth tld to owner ');
+  const tx = await root.setSubnodeOwner('0x' + keccak256('eth'), owner)
```

Comment:
Please see [here](https://github.com/ensdomains/ens-contracts/pull/127/files#diff-9886e68f1164caf4c42eeb9eb8e1572d478b9f15c499924cb1eef74085f93be5R41) for the correct way to do this. Because we use an OwnedResolver, this set of transactions isn't required.